app/server/lib/Permit.ts

```diff
@@ -32,6 +32,7 @@ export interface Permit {
   sessionId?: string;   // A particular session.
   url?: string;         // A particular url.
   action?: string;      // A string denoting what kind of action the permit applies to.
+  codeVerifier?: string;
```

Comment:
I think it's ok to remove this now, right?
```suggestion
```

Reply:
Good catch, thanks @georgegevoian!